CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Refinery Changelog
 
+## 2.9.5 2025-04-24
+
+### Features
+
+- feat: add experimental support for opamp (#1528) | [Alex Boten](https://github.com/AlexBoten)
```

Comment:
May be worth adding an extra comment here reiterating that it is experimental and not intended for use

Reply:
I will add it in the release note. I don't think we typically add extra content in changelog. I could be wrong tho

Reply:
We don't typically, but for folks who aren't aware of a separate release notes file (changelog is pretty standard), it seems worthwhile to point it out just in case.

Reply:
Maybe I have the wrong understanding there. I thought Change log is simply the list of changes introduced in a release. 

I added a note at the top section to specifically call out this.

Reply:
> I thought Change log is simply the list of changes introduced in a release.

That is true. And in this repo it seems to serve only as that purpose. But for many repos in OSS it is the only source of truth for changes and people look only at that file, so if there's something critical to be aware of, it's useful to put it in here just in case. Thanks for updating!